src/components/Tab/Tab.tsx

```diff
@@ -33,7 +35,8 @@ function Tab(props: TabProps) {
     isSubTab = false,
     navigateTo,
     info,
-    hasIncompleteFields
+    hasValidIndicator = false,
+    unfilledFields
```

Comment:
The component cares only about the length of the `unfilledFields` and not the content; let's pass only the length

Reply:
I'm planning to add some more content later on which does depend on the content, so will need it